### PR TITLE
[Geant4] Update to version 10.6.9

### PR DESCRIPTION
**Pull request by mrodozov**
please test

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: vecgeom-fix-vector.patch [hunk 1]
```diff
@@ -1,13 +0,0 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 6f5833f..d4bfd51 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -80,7 +80,7 @@ option(USE_CACHED_TRANSFORMATIONS "Use cached transformations in navigation stat
- option(FAST_MATH "Enable the -ffast-math compiler option in Release builds" OFF)
- 
- if(CMAKE_SYSTEM_PROCESSOR MATCHES "(i686|x86_64)")
--  set(VECGEOM_ARCH sse2 sse3 ssse3 sse4.1 sse4.2 avx avx2 mic mic_avx512 native empty)
-+  set(VECGEOM_ARCH sse2 sse3 ssse3 sse4.1 sse4.2 avx avx2 mic mic_avx512 native empty arch=nehalem arch=sandybridge  arch=haswell)
```
Comment (smuzaffar) [inline vecgeom-fix-vector.patch:10]: why this was completely dropped? We should patch vecgeom (https://gitlab.cern.ch/VecGeom/VecGeom/-/blob/v1.1.7/CMakeLists.txt#L88) to add `arch=nehalem arch=sandybridge  arch=haswell` support

Comment (mrodozov) [reply vecgeom-fix-vector.patch:10]: It was the https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_2_X/master/vecgeom-add-ppc64-cmake-fix.patch 
I thought I'm removing. This and https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_2_X/master/vecgeom-uninit-fix.patch should be removed from master

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
@civanch any concerns about the big number of differences in the comparison ?

**Pull request merged**